src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,81 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  pages: number[];
+  currentPage: number;
+  setCurrentPage: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  pages,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const lastPage = pages[pages.length - 1];
+  const canMovePrev = currentPage > 1;
+  const canMoveNext = currentPage < lastPage;
+
+  function onPrev() {
+    if (canMovePrev) {
+      setCurrentPage(currentPage - 1);
```

Comment:
pls use [prevState](https://blog.greenroots.info/react-hook-usestate-lazy-initialization-previous-state#heading-how-to-get-the-previous-state-data-and-use-it) for such cases